src/packages/shared-utils/package-actions/rules.ts

```diff
@@ -44,11 +44,12 @@ const arWithdrawRaiResponse: ActionRule = {
     checker.hasEnabledRaiWithdraw &&
     isStateUser(user),
 };
-
 const arWithdrawPackage: ActionRule = {
   action: Action.WITHDRAW_PACKAGE,
   check: (checker, user) =>
-    checker.isInActivePendingStatus && isStateUser(user),
+    !checker.hasStatus([SEATOOL_STATUS.APPROVED, SEATOOL_STATUS.DISAPPROVED, SEATOOL_STATUS.WITHDRAWN]) &&
+    checker.isInActivePendingStatus
```

Comment:
This is duplicative logic, I believe. It's saying:

```
NOT in a final disposition status
AND in an active pending status
```

But because status is a single value, the inclusive logic of `in an active pending status` negates any need for exclusive logic like `not in a final disposition status`.